scrabbler/player.py

```diff
@@ -16,12 +16,14 @@ def __init__ (self, midgame_strategy, endgame_strategy=None, name=None):
 
         self.name = name # might be useful for plotting during analysis
 
-    def choose_move(self, is_endgame, game):
+    def choose_move(self, is_endgame, game, other_score, other_rack, dictionary):
```

Comment:
I think you should make a `ABPruningPlayer` class that inherits from this `Player` class and overrides the `choose_move` method, instead of directly altering this method (I create a `DQNPlayer` class in my branch that you can look at for reference).